### ISSUE TITLE
Update readme.md with the current plugins

### ISSUE THREAD

**Issue by huan** [wechaty/python-wechaty-plugin-contrib#10]
Currently, we have already serval plugins but they are missing the introduction/docs in the README.md, which means that the users can not learn them for now.

We should fix it by adding related docs to readme.

**Comment by wj-Mcat** [comment]
Yes, there should be docs for plugins. I absolutely agree it and I think the full docs will be python-wechaty doc website.

Let's make it in to-do list.
